### PR TITLE
fixed issue of mypy for test_ujson

### PR DESCRIPTION
**Pull request by iamrajhans**
- [ ] xref #28926
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/io/json/test_ujson.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 try:
     import json
 except ImportError:
-    import simplejson as json
+    import simplejson as json  # type: ignore
```
Comment (simonjayhawkins) [inline pandas/tests/io/json/test_ujson.py]: this is a known mypy error, so `# type: ignore` is fine here, but can you add a comment. i.e.

```python
# https://github.com/python/mypy/issues/1153
# error: Name 'json' already defined (by an import)
```

**Review by simonjayhawkins — CHANGES_REQUESTED** [review]
@iamrajhans Thanks for the PR. change looks good.

File: pandas/tests/io/json/test_ujson.py (hunk 1 above)
Comment (iamrajhans) [reply pandas/tests/io/json/test_ujson.py]: sure, will add those comments

**Review by simonjayhawkins — APPROVED** [review]
@iamrajhans lgtm @WillAyd

File: pandas/tests/io/json/test_ujson.py [hunk 2]
```diff
@@ -1,7 +1,10 @@
 try:
```
Comment (WillAyd) [inline pandas/tests/io/json/test_ujson.py]: Can you get rid of the try...except altogether? This looks like legacy code - `import json` should work dating back to at least Python 2.6

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/tests/io/json/test_ujson.py (hunk 2 above)
Comment (iamrajhans) [reply pandas/tests/io/json/test_ujson.py]: sure, will do it

Comment (iamrajhans) [reply pandas/tests/io/json/test_ujson.py]: @WillAyd I have done the changes, please review

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @iamrajhans

**Pull request merged**